src/sqswatcher/sqswatcher.py

```diff
@@ -203,9 +221,9 @@ def _requeue_message(queue, message):
 
 
 def _retrieve_all_sqs_messages(queue, max_processed_messages):
-    log.info("Retrieving messages from SQS queue")
```

Comment:
Rather than removing this I would parametrize this log message with the queue name as done below